lib/routes/elecfans/article.ts

```diff
@@ -0,0 +1,64 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/article/:atype',
+    categories: ['programming'],
+    example: '/article/special',
+    parameters: { atype: '需获取文章的类别' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '文章',
+    radar: [
+        {
+            source: ['www.elecfans.com'],
+        },
+    ],
+    maintainers: ['tian051011'],
+    handler: async (ctx) => {
+        const { atype = 'special' } = ctx.req.param();
```

Comment:
https://github.com/DIYgod/RSSHub/blob/622390ac41c4046905c4e37ca361d9c31b33ffa6/lib/routes/elecfans/article.ts#L8 specify `atype` is a [compulsory parameter](https://hono.dev/docs/api/routing#path-parameter) which does not expect any default value